rmgpy/rmg/mainTest.py

```diff
@@ -35,6 +35,8 @@
 
 from nose.plugins.attrib import attr
 
+import pandas as pd
```

Comment:
Let's add pandas to the requirements file, environment file, and the conda meta recipe file so that this dependency is explicit